pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "dictionaryutils"
+version = "3.0.3"
+description = "Python wrapper and metaschema for datadictionary."
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+
+[tool.poetry.dev-dependencies]
+nose = ">=1.3.7<2.0.0"
```

Comment:
after testing with fence i see that ">=1<2" doesn't work, you need ">=1,<2" ⚠️

Reply:
alternatively, it looks like "^1.3.7" will automatically mean ">=1.3.7<2.0.0"
https://python-poetry.org/docs/versions/#caret-requirements